bin/storybook-default-config.js

```diff
@@ -0,0 +1,37 @@
+const { dirname } = require( 'path' );
+
+// require.resolve returns a path to a file in /dist. To get to the root of the
+// package, we need to go up the directory tree twice.
+const toPath = ( module ) => dirname( dirname( require.resolve( module ) ) );
```

Comment:
`require.resolve( module+"/package.json" )` is a safer approach, as it will always resolve to the root of the package.

Reply:
cool!